modules/test/playwright/tests/dynamic-data-mapping-form-web/formSubmission.spec.ts

```diff
@@ -8,13 +8,28 @@ import path from 'path';
 
 import {formsPagesTest} from '../../fixtures/formsPagesTest';
 import {loginTest} from '../../fixtures/loginTest';
+import {virtualInstancesPagesTest} from '../../fixtures/virtualInstancesPagesTest';
+import {liferayConfig} from '../../liferay.config';
+import {FormBuilderPage} from '../../pages/dynamic-data-mapping-form-web/FormBuilderPage';
+import {FormsPage} from '../../pages/dynamic-data-mapping-form-web/FormsPage';
+import performLogin from '../../utils/performLogin';
 import {deleteItems} from './utils/deleteItems';
 
-export const test = mergeTests(loginTest(), formsPagesTest);
+export const test = mergeTests(
+	loginTest(),
+	formsPagesTest,
+	virtualInstancesPagesTest
+);
 
 let hasDataProvider: boolean = false;
 
-test.afterEach(async ({formsPage, page}) => {
+let hasVirtualInstance: boolean = false;
+
+const DEFAULT_VIRTUAL_INSTANCE_NAME = 'www.able.com';
+
+const defaultBaseUrl = liferayConfig.environment.baseUrl;
```

Comment:
I could see that these two variables were named based on similar examples, legal 😄